data/tlg0732/tlg012/tlg0732.tlg012.1st1K-grc1.xml

```diff
@@ -0,0 +1,6021 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader xml:lang="eng">
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="grc">Ἀπορίαι καὶ λύσεις [Sp.]</title>
```

Comment:
Technically, the language here should probably be "mul", since it has "[Sp.]" in it, which is not Greek.